### ISSUE TITLE
Jackson 2.9 -> 2.10 at Dropwizard 1.3.x

### ISSUE THREAD

**Issue by vkravets** [dropwizard/dropwizard#3629]
Will Jackson update to 2.10 for DW 1.3.x? Since jackson 2.9 branch will be closed at the end of December

**Comment by joschi** [comment]
@vkravets No, Dropwizard 1.3.x won't be updated to Jackson 2.10.x or any newer version because of potentially backward-incompatible changes.

Instead, the Dropwizard 1.3.x version line won't be maintained anymore.

See also:
https://groups.google.com/g/dropwizard-user/c/6gpDMuSb4_Y

**Comment by vkravets** [comment]
Ahh, I see. In this case will DW 1.3.x include last version jackson 2.9.10.8? It should be released as I understood today...
It will be last version before end of support 2.9 branch

**Comment by joschi** [comment]
> In this case will DW 1.3.x include last version jackson 2.9.10.8?

Yes, definitely. We will upgrade to the latest release of Jackson 2.9.x once it has been released.

**Comment by vkravets** [comment]
Great, thanks. Close in this case this issue.

Happy New Year! 🎉 🍾
